pkgs/development/python-modules/solo-python/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, buildPythonPackage, fetchPypi
+, click, ecdsa, fido2, intelhex, pyserial, pyusb, requests}:
+
+ buildPythonPackage rec {
+  pname = "solo-python";
+  version = "0.0.15";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "12s5lv2n02an8nscspv8s1gknh593l7mqljqvxpnbkzfjf6icfvc";
+  };
+
+  propagatedBuildInputs = [
+    click
+    ecdsa
+    fido2 
```

Comment:
```suggestion
    fido2
```

Reply:
Is changed locally, will be pushed once the failing test is solved.